File: 8kyu/find-the-diff-in-age-between-oldest-and-youngest/find-the-diff-in-age-between-oldest-and-youngest.js
/* 
PREP:
- input is array
- find min age
- find max age
- calc the difference for oldest and youngest
- return array [youngest, oldest, the difference]
*/

function differenceInAges(ages) {
	const youngest = Math.min(...ages);
	const oldest = Math.max(...ages);
	return [youngest, oldest, oldest - youngest];
}

File: 8kyu/find-the-diff-in-age-between-oldest-and-youngest/find-the-diff-in-age-between-oldest-and-youngest.py
def difference_in_ages(ages):
  youngest = min(ages)
  oldest = max(ages)
  return (youngest, oldest, oldest - youngest)

print(difference_in_ages([16, 22, 31, 44, 3, 38, 27, 41, 88]))
print(difference_in_ages([5, 8, 72, 98, 41, 16, 55]))
print(difference_in_ages([57, 99, 14, 32]))